log/syslog_formatter.go

```diff
@@ -0,0 +1,81 @@
+package log
+
+import (
+	"fmt"
+	"log/syslog"
+	"os"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var ceeTag = []byte("@cee:")
+
+type syslogger struct {
+	wrap logrus.Formatter
+	out  *syslog.Writer
+}
+
+func newSyslogger(appname string, level string, fmter logrus.Formatter) (*syslogger, error) {
+	priority := getLocal(level)
+	out, err := syslog.New(priority, appname)
+	return &syslogger{
+		out:  out,
+		wrap: fmter,
+	}, err
+}
+
+func getLocal(level string) syslog.Priority {
+	switch level {
+	case "0":
+		return syslog.LOG_LOCAL0
+	case "1":
+		return syslog.LOG_LOCAL1
+	case "2":
+		return syslog.LOG_LOCAL2
+	case "3":
+		return syslog.LOG_LOCAL3
+	case "4":
+		return syslog.LOG_LOCAL4
+	case "5":
+		return syslog.LOG_LOCAL5
+	case "6":
+		return syslog.LOG_LOCAL6
+	case "7":
+		return syslog.LOG_LOCAL7
+	}
+	return syslog.LOG_LOCAL0
```

Comment:
@mattkanwisher So I was thinking about this after the introduction of the `switch Opague`. It might be a good idea to not return a default value for all unknown levels, but to return an error. That way a user will have immediate feedback what's wrong with `local=local7` instead of trying to understand why their log messages are delivered using the wrong facility. What do you think?